autometrics-macros/src/lib.rs

```diff
@@ -112,6 +112,10 @@ fn instrument_function(args: &AutometricsArgs, item: ItemFn) -> Result<TokenStre
         quote! { None }
     };
 
+    let service_name = quote! {
+        autometrics::__private::service_name(env!("CARGO_PKG_NAME"))
```

Comment:
If I have multiple engineers with instanced environments, should the instanced service names be handled via this, or is there a better place for that?  E.g. if I have a `loadbalancer` service and I have an instanced environment for a PR where I update said loadbalancer, should this be set to the branch name::service of `akesling-update-loadbalancer::loadbalancer` or something similar?

Reply:
This is setting the `service.name` either based on runtime environment variables (`AUTOMETRICS_SERVICE_NAME` or `OTEL_SERVICE_NAME`) or it will fall back to the package name defined in the `Cargo.toml`. I think in the case of a load balancer you would just use the runtime environment variables to set it to whatever makes sense for your deployment configuration.